tests/test_offchain.py

```diff
@@ -105,6 +105,7 @@ def test_prepare_transfer_transaction(signed_alice_transaction, recipients):
     ['CT6nWhSyE7dF2znpx3vwXuceSrmeMy9ChBfi9U92HMSP'],
 ))
 def test_fulfill_transaction(alice_transaction, alice_sk):
+    from sha3 import sha3_256
```

Comment:
same as above, we can import this at the top, as sha3 is an external lib, and is not under test

as an aside sha3 is now in the standard lib for Python 3.6+: https://docs.python.org/3/library/hashlib.html#hash-algorithms